doc/releases/migration-guide-4.0.rst

```diff
@@ -30,6 +30,9 @@ Modules
 Mbed TLS
 ========
 
+* The Kconfig options ``CONFIG_MBEDTLS_TLS_VERSION_1_0`` and ``CONFIG_MBEDTLS_TLS_VERSION_1_1``
+  have been removed because Mbed TLS has removed support for TLS 1.0 and 1.1 since a while already. (:github:`76833`)
```

Comment:
```suggestion
  have been removed because Mbed TLS has removed support for TLS 1.0 and 1.1 since a while already. (:github:`76833`)
```
```suggestion
  have been removed because Mbed TLS has removed support for TLS 1.0 and 1.1 in Mbed TLS v3.0.0.
```